sql/dml_select.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Select 语句
+category: user guide
+---
+
+# Select 语句
+
+Select 语句用于从数据库中查询数据。
+
+## 语法定义
+
+```sql
+SELECT
+    [ALL | DISTINCT | DISTINCTROW ]
+      [HIGH_PRIORITY]
+      [SQL_CACHE | SQL_NO_CACHE] [SQL_CALC_FOUND_ROWS]
+    select_expr [, select_expr ...]
+    [FROM table_references
+    [WHERE where_condition]
+    [GROUP BY {col_name | expr | position}
+      [ASC | DESC], ...]
+    [HAVING where_condition]
+    [ORDER BY {col_name | expr | position}
+      [ASC | DESC], ...]
+    [LIMIT {[offset,] row_count | row_count OFFSET offset}]
+    [FOR UPDATE | LOCK IN SHARE MODE]]
+```
+
+## 语法元素说明
+
+|语法元素 | 说明|
+|-------- | -----------|
+|ALL、DISTINCT、DISTINCTROW | 查询结果集中可能会包含重复值。指定 DISTINCT/DISTINCTROW 则在查询结果中过滤掉重复的行；指定 ALL 则列出所有的行。默认为 ALL。|
```

Comment:
Could you please add some back-ticks around these elements to make it look like an inline code? Or this will be parsed as markdown tags when converted to html pages on our website.